src/components/ModalButtonComponent.js

```diff
@@ -29,6 +31,16 @@ class ModalButtonComponent extends React.PureComponent {
 				>
 					{modalName}
 				</button>
+				{this.state.showModal ? <Modal
```

Comment:
This is pretty close to how it worked when I first implemented the modal, however, there is a bug in Webkit that prevents fixed positioned divs from displaying properly when they're a child of another fixed position div.

You can check out [this stackoverflow](https://stackoverflow.com/questions/35953770/css-parent-div-hiding-fixed-position-child-modal-in-safari) to see my fiddle example with no one finding a solution.

While I realize that other solution is not as easy to understand, as we're loading the modal into redux state, it does have a few other benefits like avoiding that webkit bug and also prevents there from being multiple modals open at once.

Reply:
I just had one of my friends with a mac try this for me and it looks ok.  Looking at the styling, there are no two fixed things inside each other.  I had him test armor mods, I'll very the other modals now though to be safe.

Reply:
The summary reddit export is actually the modal inside a fixed positioned element where the bug appears. Sorry I should have noted that.